playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -261,20 +264,38 @@ def logs_enricher(event: PodEvent, params: LogEnricherParams):
             )
         return
 
+    container: Optional[str] = None
```

Comment:
I think it would be more readable to have:
```
container: str = pod.spec.containers[0].name
```
And to remove the last `elif` clause